docs/src/main/asciidoc/datasource.adoc

```diff
@@ -32,7 +32,7 @@ Additionally, refer to the Quarkus xref:hibernate-orm.adoc[Hibernate ORM] guide
 
 == Get started with configuring `datasources` in Quarkus
 
-For rapid configuration of datasources, this section offers a brief overview and code samples for testing and utilization, suitable for users familiar with the fundamentals.
+To quickly set up datasources, this section provides a concise overview and code samples for testing and usage, suitable for users with a basic understanding of the fundamentals.
```

Comment:
The note above says 

> In text, use "data source"

Would be good to adhere to it :)

Reply:
Patching :)